contracts/libs/gov-pool/GovPoolRewards.sol

```diff
@@ -33,6 +34,12 @@ library GovPoolRewards {
 
         uint256 rewards = pendingRewards[proposalId][msg.sender];
 
+        address nftMultiplier = IGovPool(address(this)).nftMultiplier();
+
+        if (nftMultiplier != address(0)) {
+            rewards = IERC721Multiplier(nftMultiplier).multiplyRewards(msg.sender, rewards);
```

Comment:
Let's make this function add the NFT rewards to the existing rewards. `rewards += nftMultiplier.getExtraRewards(...)`